scripts/appConfig.ts

```diff
@@ -12,6 +12,14 @@ if (appConfig.shortTimeFormat == null) {
     appConfig.shortTimeFormat = 'HH:mm'; // 24h format
 }
 
+if (appConfig.view.dateformat == null) {
+    appConfig.view.dateformat = 'MM/DD'; // 24h format
```

Comment:
your comment doesn't make sense since days of the month don't come in 24h format